swap_meet/electronics.py

```diff
@@ -1,2 +1,12 @@
-class Electronics:
-    pass
+
+from .item import Item
+
+class Electronics(Item):
+    def __init__(self, id = None, condition = None, type = None):
```

Comment:
Per the README, the default value for condition should be `0` instead of `None`. Think you have the default value for condition set  to None for all the classes so they would all need to be updated to 0.